0000-plugins.md

```diff
@@ -0,0 +1,169 @@
+- Feature Name: plugins
+- Start Date: 2024-12-02
+- RFC PR: [LemmyNet/rfcs#0000](https://github.com/LemmyNet/rfcs/pull/0008)
+- Lemmy Issue: [LemmyNet/lemmy#0000](https://github.com/LemmyNet/lemmy/issues/3562)
+
+# Summary
+
+Plugins will make Lemmy more flexible, and allow implementing features which are too niche for merging into Lemmy core. It will allow Lemmy to focus on core features, while allowing outside developers to contribute new features without using Rust.
+
+# Motivation
+
+TODO
```

Comment:
Not sure what this section is good for, its practically the same as summary.

Reply:
I'm not into the terminology you use in the repository, but a summary may provide a TL;DR about the **how** (implementation and design), while a motivation would tell us more about the **why** (reasoning, value proposition, business case, user story).

Reply:
Its based on [Rust RFCs](https://github.com/rust-lang/rfcs). Maybe we should change this part in the template as our RFCs arent that extensive.

Reply:
I don't see a contradiction between that and what I have told above
* https://github.com/rust-lang/rfcs/blob/master/0000-template.md#motivation

Reply:
Updated